lepton/image.go

```diff
@@ -249,62 +256,82 @@ func (bc dummy) Write(p []byte) (int, error) {
 	return len(p), nil
 }
 
-func DownloadBootImages() error {
```

Comment:
Dont delete this function, it's used else where.

Reply:
I didn't delete this function. I just extended it to make sure that `OpsDir` exists and it's name is the same `DownloadBootImages`. I executed tests and checked for use of that function - everything is alright

Reply:
You renamed it right? What I was telling is that it's used outside of this repository. So you won't see any test failure, remember this is a library shared across projects

Reply:
Got it, returned back function with it's parameters